Dz_4_5/src/bohdan/homework/com/Fighters/Base/Actions/ReturnMyDamage.java

```diff
@@ -0,0 +1,17 @@
+package bohdan.homework.com.Fighters.Base.Actions;
+
+import bohdan.homework.com.Fighters.Base.ArenaFighter;
+
+public interface ReturnMyDamage {
+
+    /**
+     *
+     * @param arenaFighterOur  Our Fighter
+     * @param arenaFighterEnemy  Enemy Fighter
+     * @return damage our fighter
+     */
+    default int getDamageVampirism(ArenaFighter arenaFighterOur, ArenaFighter arenaFighterEnemy) {
+        return arenaFighterOur.getDamage() - (arenaFighterOur.getDamage() * (int)arenaFighterEnemy.getDefense());
```

Comment:
vampirism has to be counted by facticaly GIVEN damage, (knights could not receive damage at all)